server/server.js

```diff
@@ -170,6 +183,36 @@ app.post('/api/tripUpdate', (req, res) => {
     });
 });
 
+app.post('/api/resetPassword', (req, res) => {
+    const newPassword = uuid();
+
+    encryptPassword(newPassword, (err, encrypted) => {
+        if (err) return res.json({ success: false, message: err });
+
+        User.findOneAndUpdate({ email: req.body.email }, { password: encrypted }, (err, user) => {
+            if (err) return res.json({ success: false, message: err });
+            if (!user) return res.json({ success: false, message: 'Request failed, email not found'});
+            
+            const mailOptions = {
+                from: `"Admin TripReview" <${adminMail}>`,
+                to: req.body.email,
+                subject: 'Reset Password',
+                text: `Your new password is ${newPassword}`
```

Comment:
In the future I have to change this to some html template with more meaningful info to make the page look more professional